libs/sdk-core/src/invoice.rs

```diff
@@ -189,6 +190,11 @@ pub fn add_lsp_routing_hints(
 
 /// Parse a BOLT11 payment request and return a structure contains the parsed fields.
 pub fn parse_invoice(bolt11: &str) -> InvoiceResult<LNInvoice> {
+    if bolt11.is_empty() {
```

Comment:
```suggestion
    if bolt11.trim().is_empty() {
```

Calling [trim()](https://doc.rust-lang.org/std/primitive.str.html#method.trim) also covers strings made up of only whitespace (tabs, space, etc).

Reply:
👍 done